FormalConjectures/Util/Linters/CopyrightLinter.lean

```diff
@@ -0,0 +1,54 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import Mathlib.Tactic.Linter.Header
+
+open Lean Elab Meta Command Syntax
+
+def correctCopyrightHeader : String :=
+"/-
+Copyright 2025 The Formal Conjectures Authors.
```

Comment:
This will break down when people start using "2026" here next year?

Either fix now, or leave a TODO (which should ideally be addressed in the next 6 months)

Reply:
Good point! Added a note